core/src/main/java/com/novoda/merlin/Merlin.java

```diff
@@ -10,11 +10,11 @@
 public class Merlin {
 
     private final MerlinServiceBinder merlinServiceBinder;
-    private final Registerer registerer;
+    private final Registrar registrar;
```

Comment:
Yea this is what I said to @joetimmins 😄  it's a pretty unknown word for most people 😆  but it is more accurate.